p4-16/spec/P4-16-spec.mdk

```diff
@@ -1773,6 +1773,7 @@ baseType
     : BOOL
     | ERROR
     | BIT
+    | INT
```

Comment:
Minor: the `int` type should be added to the list "P4 supports the following built-in base types" above...